codeov.yml

```diff
@@ -0,0 +1,10 @@
+comment:
+  layout: "header, diff, tree"
+
+codecov:
```

Comment:
We may not need this token. Codecov doesn't need token to be provided for Travis